libs/partners/ai21/tests/integration_tests/test_llms.py

```diff
@@ -0,0 +1,124 @@
+"""Test AI21LLM llm."""
+
+import pytest
+from ai21.models import Penalty
+
+from langchain_ai21.llms import AI21
+
+
+def _generate_llm_client_parameters() -> AI21:
```

Comment:
```suggestion
def _generate_llm() -> AI21:
```
if you could add a docstring of what these params are testing vs. the default `AI21LLM()` that would be great!

Reply:
Done